lang/pc/comp/LLlex.c

```diff
@@ -153,9 +153,7 @@ SkipComment()
 	}
 }
 
-STATIC struct string *
-GetString( delim )
-register int delim;
+static struct string *GetString(register int delim)
```

Comment:
This is not a request for a change, just a comment: no compiler, not even the ACK, honours `register` any more. Might as well delete 'em if they're in the way.